packages/web3-utils/src/socket_provider.ts

```diff
@@ -216,11 +216,12 @@ export abstract class SocketProvider<
 	 * @param data - The data to be sent to the server
 	 */
 	public disconnect(code?: number, data?: string): void {
+		const disconnectCode = code ?? 1000;
```

Comment:
Where this 1000 came from? Could we please reference a resource to its origin?
And I think it is ideally to put this 1000 in a constant and export it from `web3-types`.

Reply:
ok I will add this as a comment https://github.com/Luka967/websocket-close-codes